vms/platformvm/state/state.go

```diff
@@ -446,11 +448,11 @@ type state struct {
 	utxoCache     cache.Cacher[ids.ID, *avax.UTXO] // UTXO ID -> *UTXO. If the *UTXO is nil the UTXO doesn't exist
 
 	// Metadata section
-	chainTime, latestComittedChainTime                  time.Time
```

Comment:
@danlaine I could backport `latestComittedChainTime` and `latestCommittedLastAcceptedBlkID` to `pchain_merklDB` branch.
I cannot recall why I kept these attributes in the first place. Looks to me now they are not really relevant, but would like your opinion on it